### ISSUE TITLE
Deploy a Docker instance to one of a 'Fleet' of Ships

### ISSUE THREAD

**Issue by monokal** [signalfuse/maestro-ng#52]
Rather than specifying a particular Ship on which to deploy a Docker instance, it would be useful to be able to deploy to a given 'Fleet' (pool) of Ships, with Maestro-ng working out which Ship in the Fleet to deploy to using a simple round-robin or balancing algorithm.

As a future 'nice-to-have' it would also be nice for Maestro-ng to watch for Serf MemberJoin and MemberLeave events as to dynamically construct the Fleet, although I appreciate this may be well out of the scope of the project.

Basically, I want to say "Deploy to the fleet!" without caring which ship it deploys to.
